definitions/npm/awilix_v2.x.x/flow_v0.56.x-/awilix_v2.x.x.js

```diff
@@ -1,5 +1,5 @@
 declare module "awilix" {
-  declare module.exports: {
+  declare module .exports: {
```

Comment:
Pretty sure this is a typo

Reply:
Jetbrains does that automatically and they haven't fixed it yet, edited and pushed, sorry